transifex/native/tools/migrations/review.py

```diff
@@ -26,45 +35,19 @@
 # The reviewer has decided to mark a specific file for later review
 REVIEW_MARK_FILE = 6
 
+# The reviewer has decided to exit the migration, skipping remaining changes
+REVIEW_EXIT = 7
+
 # When the user is prompted to review a change (file or string)
 # these are the valid responses
 ACCEPT_CHOICE = 'A'  # accept the change
 ACCEPT_ALL_CHOICE = 'AA'  # accept all changes
 REJECT_CHOICE = 'R'  # reject the change
-REJECT_ALL_CHOICE = 'RR'  # reject all change
+REJECT_ALL_CHOICE = 'RR'  # reject all changes
+EXIT_CHOICE = 'X'  # exit the migration completely
 MARK_CHOICE = 'M'  # mark for later review (inside the file, if saved)
 PRINT_CHAR = 'P'  # print the diff
 
```

Comment:
Minor: maybe grouping all these choices? Probably a Class or a dictionary?

Reply:
A dictionary is not convenient because you would lose the direct reference by variable. A class would be OK but so is the current implementation, I think I'll leave it as is.